manager/src/test/resources/application.properties

```diff
@@ -1,30 +1,7 @@
-# Database
-quarkus.datasource.db-kind=h2
```

Comment:
You should be able to remove [`quarkus-jdbc-h2`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/manager/pom.xml#L80-L84) dependency

Reply:
Removed :+1: